orc8r/cloud/go/services/analytics/analytics/main.go

```diff
@@ -14,43 +14,25 @@
 package main
 
 import (
-	"flag"
 	"magma/orc8r/cloud/go/services/analytics"
 
 	"magma/orc8r/cloud/go/orc8r"
 	"magma/orc8r/cloud/go/service"
-	"magma/orc8r/lib/go/service/config"
 
 	"github.com/golang/glog"
 )
 
-const (
-	defaultAnalysisSchedule = "*/1 * * * *" // Every 1 minute
-)
-
-//ServiceMain - entrypoint for analytics service
 func main() {
-	flag.Parse()
-
 	// Create the service
 	srv, err := service.NewOrchestratorService(orc8r.ModuleName, analytics.ServiceName)
 	if err != nil {
-		glog.Fatalf("Error creating Analytics service: %s", err)
+		glog.Fatalf("Failed running Analytics service: %v", err)
```

Comment:
In my cursive search, I see a lot of occurrences for Error creating. I will keep it this way. But I agree with your point. Keeping it consistent helps. For addressing this, usually standardized log messages are created and it will encapsulate the a sub message. The standardized log messages are later used by customers to build tools around recognizing class of failures. We could perhaps move to that instead of standardizing through documentation. 
For e.g. https://www.cisco.com/en/US/docs/security/asa/asa80/system/message/logmsgs.pdf

https://github.com/magma/magma/search?l=Go&q=%22Error+creating%22